Java/Java.sublime-syntax

```diff
@@ -568,8 +572,7 @@ contexts:
           pop: true
         - include: illegal-parens-terminators
         - include: code
-  for-loop-parens:
-    - meta_scope: for-loop
+  declaration-statement-parens:
     - match: \(
       scope: punctuation.section.parens.begin.java
       set:
```

Comment:
This is really a comment for line 584 but I can't seem to add one there.

You could change `include: code` to `include: code-block` and avoid declaring a final block.

Reply:
Yes, this seems to be a more correct solution. Fixed.